core/jvm/src/module-info.java

```diff
@@ -0,0 +1,7 @@
+module kotlinx.datetime {
+    requires transitive kotlin.stdlib;
+    requires transitive static kotlinx.serialization.core;
```

Comment:
Yes and no. Normally a modular application won't start if 1 of the required modules isn't on the module path.
Because of the `requires static` an application will start anyway, ignoring the 'missing' module.
Obviously if you want to use the datetime serializers you will have to explicitly require `kotlinx.serialization.core` anyway because all the serializer interfaces and logic is in there.